src/main/java/com/codecool/dungeoncrawl/data/GameMap.java

```diff
@@ -12,6 +14,9 @@ public class GameMap {
 
     private Skeleton skeleton;
 
+    private Ghost ghost;
+    private Wizard wizard;
```

Comment:
Enemies could be bundled together ;) also they should. If you want to rely on Dependency inversion (inversion is not just about injection!) which is about depending on abstraction instead of concrete classes